commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AuditLogging.java

```diff
@@ -0,0 +1,132 @@
+package org.wildfly.extras.creaper.commands.auditlog;
+
+/**
+ * A slightly more convenient way to create commands for Audit Logging. All parameters in this class's methods must be
+ * specified.
+ */
+public final class AuditLogging {
+
+    private AuditLogging() {}
+
+    public static AuditLogDestinationHandler destinationHandler() {
+        return AuditLogDestinationHandler.INSTANCE;
+    }
+
+    public static AuditLogHandler auditLogHandler() {
+        return AuditLogHandler.INSTANCE;
+    }
+
+// ---
```

Comment:
Also appears on other places in this class.

The `// ---` comment is my personal habit, I don't require anyone to use it.